content/influxdb/cloud/reference/release-notes/cloud-updates.md

```diff
@@ -14,6 +14,34 @@ aliases:
 InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
 To find information about the latest Flux updates in InfluxDB Cloud, see [Flux release notes](/influxdb/cloud/reference/release-notes/flux/).
 
+## November 2021
+
+- [Remove Website demo data](#remove-website-demo-data)
+- [Add sample data buckets to Notebooks UI](#add-sample-data-buckets-to-notebooks-ui)
+- [Add ability to share notebooks](#add-ability-to-share-notebooks)
+- [Update Tokens UI](#update-tokens-ui)
+
+### Remove Website Demo Data
+
+To improve user experience and consolidate system-delivered data, we removed the Website Demo Data bucket in Data Explorer. If you're looking for a data to get started with in InfluxDB Cloud, check out sample data buckets, which support a variety of use cases.
+
+### Add sample data buckets to Notebooks UI
+
+Sample data buckets provide a way to explore InfluxDB without ingesting your own data. Now, you can add sample data buckets directly into your notebooks:
+
+- Air Sensor Data
+- Coinbase bitcoin price
+- NOAA National Buoy Data
+- USGS Earthquakes
+
+### Add ability to share notebooks
+
+Add ability to [share a notebook](/influxdb/cloud/notebooks/manage-notebooks/#share-a-notebook) in the the InfluxDB Cloud notebook UI.
```

Comment:
added docs update to _Overview of notebooks_ and _Manage notebooks_ in [this PR]( https://github.com/influxdata/docs-v2/pull/3435/files)

Reply:
PR merged